resteasy-core/src/main/java/org/jboss/resteasy/core/interception/jaxrs/PreMatchContainerRequestContext.java

```diff
@@ -368,4 +370,17 @@ public boolean startedContinuation()
    {
       return startedContinuation;
    }
+
+   private void cleanupPostResourceMethodInvokers() {
+      PostResourceMethodInvokers postResourceMethodInvokers =
+              ResteasyContext.getContextData(PostResourceMethodInvokers.class);
+      // close PostResourceMethodInvokers and clear array list
+      if (postResourceMethodInvokers != null) {
+         postResourceMethodInvokers.getInvokers().forEach(e -> e.close());
+      }
+      // finally
+      if (postResourceMethodInvokers != null) {

```

Comment:
These two lines could just be deleted. It might make sense to use a try/catch for each invoker too and log the failures. Otherwise we could end up with invokers in weird states.